experiments/pages/pageSupportRemover/README.md

```diff
@@ -0,0 +1,9 @@
+# Page Support File Remover
+
+As of Skuid release 12.2.0, page support files are no longer needed by Skuid to render in-line Javascript and CSS. This is not a breaking change and orgs with page support files will continue to work as expected. However, if users want to keep their static resources clean, this tool will attempt to find any page support files and then allow users to remove them.
+
+## Usage
+
+To use this tool, create a Skuid page in your Salesforce org with the page XML stored in this folder. Once created, click **Load all Page Support Files**, select the files you wish to be deleted, and then click **Delete Page Support Files**
```

Comment:
```suggestion
To use this tool, create a Skuid page in your Salesforce org with the page XML stored in this folder. Once created, click **Load all Page Support Files**, select the files you wish to delete, and then click **Delete Page Support Files**
```

Reply:
Actually, can we break these steps up into a numbered list? Also we need to bold that the page is a *v2 page*. I tried to create this as a v1 page and got errors.

Reply:
Also we need a step that says *Preview* the page. You can't do any of the steps from the App Composer.